app/services/task_status/dwp_outcome.rb

```diff
@@ -0,0 +1,38 @@
+module TaskStatus
+  class DWPOutcome < Base
+    def call
+      status = ValueObject.new
+
+      status.not_ready! if not_ready?
+      status.not_started! if not_started?
+      status.in_progress! if in_progress?
+      status.completed! if completed?
+
+      status
+    end
+
+  private
+
+    def not_ready?
+      true
+      # TODO: update once validator is ready
+      # !check_your_answers_validator.valid?
+    end
+
+    def not_started?
+      application.confirm_dwp_result.nil?
+      # TODO: update once CYA validator is ready
+      # check_your_answers_validator.valid? && application.confirm_dwp_result.nil?
```

Comment:
As for the other status methods below the validator will need to consider both the answer to the confirmation but also, in the event of a "No", I think it will also need to validate the dwp_overide forms to determine whether the task list item is "in progress" or "completed".